Networks/index.js

```diff
@@ -0,0 +1,101 @@
+const http = require("http");
+const UserController = require("./src/controller/userController");
+const { getReqData } = require("./src/utility/read");
+
+const server = http.createServer(async (req, res) => {
+  res.setHeader("Content-Type", "application/json");
+
+  if (req.url === "/users" && req.method === "GET") {
+    //Get All User
+    const users = await new UserController().getAllUsers();
```

Comment:
ideally, you can create an object of UserController and then use it multiple time like below,

```
const userController = new UserController();

await userController.getUsers()
await userController.getUser(id)
await userController.updateUser(id, payload);
await userController.deleteUser(id);
```